### PR TITLE
Fix image name for 5.6

### PR DESCRIPTION
**Pull request by mnagy**
This is just a tiny deduplication change, will be helpful later.

### CODE REVIEW

**Comment by mnagy** [comment]
@rhcarvalho PTAL

[test]

**Comment by rhcarvalho** [comment]
LGTM

**Comment by mnagy** [comment]
one moment, found other stuff

**Comment by mnagy** [comment]
Made changes to 5.6 Dockerfile to align with that of 5.5. It should not introduce any functional difference, but would be hell to maintain if we want to deduplicate and generate.

PTAL

File: 5.6/Dockerfile.rhel7 [hunk 1]
```diff
@@ -31,7 +31,7 @@ EXPOSE 3306
 # This image must forever use UID 27 for mysql user so our volumes are
 # safe in the future. This should *never* change, the last test is there
 # to make sure of that.
-RUN yum install -y yum-utils gettext hostname && \
+RUN yum install -y yum-utils && \
```
Comment (rhcarvalho) [inline 5.6/Dockerfile.rhel7]: hmmm this was marked for install twice :)

**Comment by rhcarvalho** [comment]
still LGTM

File: 5.6/Dockerfile.rhel7 (hunk 1 above)
Comment (mnagy) [inline 5.6/Dockerfile.rhel7]: indeed :)

**Comment by mnagy** [comment]
Thanks @rhcarvalho, will merge once test passes.

**Pull request merged**